File: Examenes/22-1/JAVA/EduSoftLP2Model/src/pe/edu/pucp/edusoftlp2/model/DocenteExtranjero.java
package pe.edu.pucp.edusoftlp2.model;
public class DocenteExtranjero extends Docente{
    private Filiacion filiacion;
    private boolean dominaEspanhol;
    private boolean dominaIngles;
    private boolean gradoRevalidadoSUNEDU;
    private boolean activo;

    public Filiacion getFiliacion() {
        return filiacion;
    }

    public void setFiliacion(Filiacion filiacion) {
        this.filiacion = filiacion;
    }

    public boolean isDominaEspanhol() {
        return dominaEspanhol;
    }

    public void setDominaEspanhol(boolean dominaEspanhol) {
        this.dominaEspanhol = dominaEspanhol;
    }

    public boolean isDominaIngles() {
        return dominaIngles;
    }

    public void setDominaIngles(boolean dominaIngles) {
        this.dominaIngles = dominaIngles;
    }

    public boolean isGradoRevalidadoSUNEDU() {
        return gradoRevalidadoSUNEDU;
    }

    public void setGradoRevalidadoSUNEDU(boolean gradoRevalidadoSUNEDU) {
        this.gradoRevalidadoSUNEDU = gradoRevalidadoSUNEDU;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }
    
    
}

File: Examenes/EX1/22-1/CSharp/EduSoftLP2/EduSoftLP2Model/DocenteExtranjero.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EduSoftLP2Model
{
    public class DocenteExtranjero : Docente
    {
        private Filiacion _filiacion;
        private bool _dominaEspanhol;
        private bool _dominaIngles;
        private bool _gradoRevalidadoSUNEDU;
        private bool _activo;

        public Filiacion Filiacion { get => _filiacion; set => _filiacion = value; }
        public bool DominaEspanhol { get => _dominaEspanhol; set => _dominaEspanhol = value; }
        public bool DominaIngles { get => _dominaIngles; set => _dominaIngles = value; }
        public bool GradoRevalidadoSUNEDU { get => _gradoRevalidadoSUNEDU; set => _gradoRevalidadoSUNEDU = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}
